integration_tests/dbt_project.yml

```diff
@@ -6,27 +6,24 @@ version: '0.1.0'
 profile: 'integration_tests' 
 
 vars:
+  apple_store_schema: apple_store_source_integration_tests
```

Comment:
The edits here are allow for the generated docs to not have the floating seed files and instead leverage the source files themselves. This makes understanding the docs we host on the github page easier to understand.